src/middleware/authentication.middleware.ts

```diff
@@ -0,0 +1,11 @@
+import { authMiddleware } from "web-security-node";
+import * as pageURLs from "../model/page.urls";
+
+const authMiddlewareConfig = {
+  accountWebUrl: "",
+  returnUrl: `${pageURLs.STRIKE_OFF_OBJECTIONS}${pageURLs.COMPANY_NUMBER}`,
+};
+
+const signInMiddleware = authMiddleware(authMiddlewareConfig);
```

Comment:
Perhaps call this authenticationMiddleware and export/import it as this. Then consistent with sessionMiddleware, below.

Reply:
will do

Reply:
done